ts/torch_handler/base_handler.py

```diff
@@ -370,7 +377,10 @@ def inference(self, data, *args, **kwargs):
             Torch Tensor : The Predicted Torch Tensor is returned in this function.
         """
         with torch.inference_mode():
-            marshalled_data = data.to(self.device)
+            if IPEX_AVAILABLE and IPEX_GPU: # IPEX GPU
+                marshalled_data = data.to("xpu")
```

Comment:
need add xpu id.

Reply:
Thank you for the suggestion. We have updated this.